base-theme/layouts/partials/external_resource_link.html

```diff
@@ -1,4 +1,4 @@
-{{- $text := default .title .text | htmlEscape | page.RenderString -}}
+{{- $text := default .title .text -}}
```

Comment:
`htmlEscape` is not required, rather it should be avoided. This is because Studio renders escaped content where required.

`RenderString` is unnecessary because markdown is rendered automatically in the relevant shortcode.